README.md

```diff
@@ -83,6 +112,25 @@ Open your browser and navigate to [http://localhost:3000](http://localhost:3000)
   - Make sure Docker Desktop is running
   - Try restarting Docker Desktop
   - Run `docker compose down` and then `make build` again
+    
+- **File Table Overflow or Too Many Open Files**: If you encounter ENFILE: file table overflow or too many open files errors, check your system’s default file limit using the following command:
```

Comment:
that shouldn't happen, if you use make script which sets up correct nvm and uses pnpm